build/BuildSteps.cs

```diff
@@ -6,29 +6,34 @@
 using System.Net;
 using System.Runtime.InteropServices;
 using System.Text.RegularExpressions;
+using System.Threading.Tasks;
 using Colors.Net;
-using Colors.Net.StringColorExtensions;
-using Microsoft.WindowsAzure.Storage;
-using Microsoft.WindowsAzure.Storage.Blob;
 using Newtonsoft.Json;
 
 namespace Build
 {
     public static class BuildSteps
     {
-        private const string Net8ArtifactNameSuffix = "_net8";
+        private const string Net6TargetFramework = "net6.0";
+        private const string DefaultTargetFramework = "net8.0";
```

Comment:
nit: maybe have these consts in a separate file just to clean this up a little (since these are .NET related consts)